ansible_runner/__main__.py

```diff
@@ -528,6 +528,33 @@ def main(sys_args=None):
         dest='resource_profiling_results_dir',
         help="Directory where profiling data files should be saved. Defaults to None (profiling_data folder under private data dir is used in this case).")
 
+    playbook_group.add_argument(
+        "--containerized",
+        dest='containerized',
+        action="store_true",
+        help="Indicates ansible task should be executed using an execution environment")
+
+    playbook_group.add_argument(
+        "--container-runtime",
+        dest="container_runtime",
+        default="podman",
+        help="Container engine to use when running an ansible task"
+    )
+
+    playbook_group.add_argument(
+        "--container-image",
+        dest="container_image",
+        default="ansible/ansible-runner",
+        help="Container image to use when running an ansible task (see --containerized)"
+    )
+
+    playbook_group.add_argument(
+        "--container-volume-mounts",
```

Comment:
I can see the purpose for volume mounts here but it may be necessary to support arbitrary container parameterization (such as resource usage, network attachment, etc). Either in the form of catchall parameterization or config file.

Reply:
+1 to that, I think trying to wrap each param we want to target for various isolation purposes will become painful to maintain quickly

Reply:
added `--container-options` as a catch-all for any remaining options the user may want to pass to runner